### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by jdent02**
Add additional controls to Specular BTDF

Add Sheen BRDF

Redo material name display

### CODE REVIEW

File: project_file_writer.py [hunk 1]
```diff
@@ -1340,6 +1354,55 @@ def __emit_diffuse_btdf(self, material, bsdf_name, scene, layer=None, node=None)
         self.__emit_parameter("transmittance_multiplier", transmittance)
         self.__close_element("bsdf")
 
+    #------------------------
+    #Write Sheen BRDF
```
Comment (dictoon) [inline project_file_writer.py]: Insert space after #

File: properties/materials.py [hunk 2]
```diff
@@ -783,6 +786,46 @@ class AppleseedMatLayerProps(bpy.types.PropertyGroup):
...
     # ---------------------------------
 
+    sheen_reflectance = bpy.props.FloatVectorProperty(name="Reflectance",
+                                                      description="Reflectance", subtype='COLOR',
+                                                      default=(0.5, 0.5, 0.5),
+                                                      min=0.0,
+                                                      max=1.0,
+                                                      update=refresh_preview)
+
+    sheen_reflectance_use_tex = bpy.props.BoolProperty(name="", description="Use a texture to influence sheen reflectance", default=False,
```
Comment (dictoon) [inline properties/materials.py]: Let's put one parameter per line (everywhere in this file)?

File: properties/materials.py [hunk 3]
```diff
@@ -811,9 +854,36 @@ class AppleseedMatLayerProps(bpy.types.PropertyGroup):
                                                    description="Specular BTDF transmittance multiplier", default=1.0, min=0.0, max=1.0,
                                                    update=refresh_preview)
 
+    spec_fresnel_multiplier = bpy.props.FloatProperty(name="Fresnel Multiplier",
+                                                  description="Fresnel multiplier",
```
Comment (dictoon) [inline properties/materials.py]: Fix indentation.

File: ui/materials.py [hunk 4]
```diff
@@ -1116,9 +1166,29 @@ def draw(self, context):
 
                     layout.prop(current_layer, "spec_btdf_trans_mult")
 
+                    # fresnel multiplier
```
Comment (dictoon) [inline ui/materials.py]: *fresnel* -> *Fresnel* (https://en.wikipedia.org/wiki/Augustin-Jean_Fresnel)

**Comment by jdent02** [comment]
I fixed the areas you specifically mentioned.  As far as the 'one parameter per line' that's going to be a major project as there are hundreds of parameters that will need to be updated.

**Pull request merged**